_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -128,47 +129,67 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86
 
 <div class="quickstart_os quickstart_os_debian10">
   <p>
-    Using Puppet 6.x is recommended, which is available from the Puppet Labs repository.
+    Using Puppet 7.x is recommended, which is available from Puppet's repository.
 
-    To use Puppet 6.x with Puppet Agent and Puppet server:
+    To use Puppet 7.x with Puppet Agent and Puppet server:
   </p>
 
 {% highlight bash %}
 sudo apt-get -y install ca-certificates
-cd /tmp && wget https://apt.puppet.com/puppet6-release-buster.deb
-sudo dpkg -i /tmp/puppet6-release-buster.deb
+cd /tmp && wget https://apt.puppet.com/puppet7-release-buster.deb
+sudo dpkg -i /tmp/puppet7-release-buster.deb
 {% endhighlight %}
 
   <p>Enable the Foreman repo:</p>
 
 {% highlight bash %}
+wget https://deb.theforeman.org/foreman.asc -O /etc/apt/trusted.gpg.d/foreman.asc
 echo "deb http://deb.theforeman.org/ buster {{page.version}}" | sudo tee /etc/apt/sources.list.d/foreman.list
 echo "deb http://deb.theforeman.org/ plugins {{page.version}}" | sudo tee -a /etc/apt/sources.list.d/foreman.list
-sudo apt-get -y install ca-certificates gpg
-wget -q https://deb.theforeman.org/pubkey.gpg -O- | sudo apt-key add -
+{% endhighlight %}
+</div>
+
+<div class="quickstart_os quickstart_os_debian11">
+  <p>
+    Using Puppet 7.x is recommended, which is available from Puppet's repository.
```

Comment:
7 is the only supported one on Debian 11, should we rephrase that?

Reply:
Something like this? I'd like to keep the text as equivalent as possible between the various distros. 
```suggestion
    Using Puppet 7.x from Puppet's repository is recommended.
```

Or, since it's the quickstart, say (the same as with the Foreman repo):
```suggestion
     Enable Puppet's 7.x repository:
```

And I do think we should write repository instead of repo on line 165.

Reply:
Yeah 👍

Reply:
Updated